implementations/rust/ockam/ockam_api/src/kafka/secure_channel_map/secure_channels.rs

```diff
@@ -22,7 +22,7 @@ impl KafkaSecureChannelControllerImpl {
 
         let secure_channel = inner
             .node_manager
-            .create_secure_channel(context, destination, None, None, None, None)
+            .create_secure_channel(context, destination, None, None, None, None, false)
```

Comment:
I think it would help the reader and help prevent future mistakes to introduce an enum here. For example:
```
enum SecureChannelType {
  KeyExchangeAndMessages,
  KeyExchangeOnly
}
```
The other alternative is to create a builder pattern which will also allow to name all those `None`s.